database/migrations/2023_06_01_124127_update_menu_items_table.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class UpdateMenuItemsTable extends Migration
+{
+    public function up()
+    {
+        Schema::table('menu_items', function (Blueprint $table) {
+            $table->string('label')->change();
+            $table->integer('page_id')->nullable()->change();
+            $table->integer('landing_page_id')->nullable()->change();
```

Comment:
Usually, foreign keys are created with `foreignId()` or `foreignIdFor()`, see https://laravel.com/docs/8.x/migrations#column-method-foreignId

Reply:
I made the change here and in the original table definition.